IS7024/Pages/LudlowGarage.cshtml

```diff
@@ -4,74 +4,60 @@
     ViewData["Title"] = "LudlowGarage";
     var events = (List<QuickType.Event>)ViewData["events"];
     var sun = ViewData["sun"];
+    string eventlocation = (string)ViewData["eventlocation"];
 }
 
-    <div class="text-center">
-        <h1 class="display-4">Upcoming Concerts at the Ludlow Garage</h1>
-        <p>Cincinnati, Ohio</p>
-        <hr>
-        <table border="1" width="90%">
-            <tr>
-                <th>Date</th>
-                <th>Artist</th>
-                <th>Status</th>
-                <th>Start Time</th>
-                <th>Sunset Time</th>
-                <th>Buy tickets</th>
-            </tr>
-            @{
-
-    foreach (QuickType.Event eventss in events)
-    {
-
-        string date = eventss.Start.Date.ToString();
-        string[] dars = date.Split('/');
-        string day = dars[0];
-        string month = dars[1];
-        string year = dars[2];
-        string[] year1 = year.Split(' ');
-        string yearclean = year1[0];
-
-
-        if (eventss.Start.Datetime != null)
-
-        {
-            string[] tars = eventss.Start.Datetime.Split('T');
-            string[] tars2 = tars[1].Split(':');
-            string mhour = tars2[0];
-            string mmin = tars2[1];
-
-            var newhour = Convert.ToInt32(mhour);
-            int realhour = newhour - 12;
-            string starttime = realhour + ":" + mmin + " PM";
-
-
-                     
+<div class="text-center">
+    <h1 class="display-4">Upcoming Concerts at the Ludlow Garage</h1>
+    @{ 
+        <h4>@eventlocation</h4>
+    }
```

Comment:
This was added so that the city and state of the venue is pulled from the JSON data for that venue instead of written in.